src/manifest.json

```diff
@@ -12,16 +14,36 @@
   },
   "content_scripts": [
     {
-      "matches": ["http://*.tiktok.com/*", "https://*.tiktok.com/*"],
-      "js": ["contentScript.bundle.js"],
-      "css": ["content.styles.css"]
+      "matches": [
+        "http://*.tiktok.com/*",
+        "https://*.tiktok.com/*"
+      ],
+      "js": [
+        "contentScript.bundle.js"
+      ],
+      "css": [
+        "content.styles.css"
+      ]
+    },
+    {
+      "matches": [
+        "https://*.youtube.com/*",
+        "http://*.youtube.com/*"
+      ],
+      "js": [
+        "ytContentScript.bundle.js"
```

Comment:
Not sure if I missed anything, but I don't see any webpack process that emitting this js file

Reply:
It’s exported from Line 32 webpack.config